gautomatch/protocols/protocol_gautomatch.py

```diff
@@ -328,6 +328,11 @@ def _pickMicrograph(self, mic, args):
 
     def _pickMicrographList(self, micList, args):
         """ Pick several micrographs at once, probably a bit more efficient."""
+        micPath = self._getMicrographDir(micList[0])
+        if len(micList) > 1:
+            micPath += ('-%06d' % micList[-1].getObjId())
+        pwutils.makePath(micPath)
```

Comment:
After picking is done, we should remove this temporary folder...to avoid accumulating extra files.